bundler/man/bundle-gem.1.txt

```diff
@@ -83,9 +83,27 @@ OPTIONS
               displayed  and  the  answer  will  be  saved in Bundler's global
               config for future bundle gem use.
 
+       --ci, --test=github, --test=travis, --test=gitlab, --test=circle
```

Comment:
Why we pass the ci value to the `--test` flag instead of the `--ci` flag?

Reply:
Sorry, that was a bad copy pasta.